util/fpga/cw310_serial.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+screen /dev/ttyACM1 115200,cs8,-ixon,-ixoff
```

Comment:
Do we really need this script? We don't have it for other boards.

Reply:
Removed.

Reply:
When we were testing, the serial port would block if not using `cs8,-ixon,-ixoff` so had added that script out of laziness. We didn't normally use `screen` as our serial terminal interface of choice however, so wasn't sure if that was somehow related to our system or not. All the docs etc used screen as that seemed to be the default in OpenTitan docs.